Spiders/spiders_practice/spiders_practice/items.py

```diff
@@ -0,0 +1,83 @@
+import scrapy
+from scrapy.loader import ItemLoader
+
+
+class ProductItem(scrapy.Item):
+    identifier = scrapy.Field()
+    base_sku = scrapy.Field()
+    url = scrapy.Field()
+    title = scrapy.Field()
+    category_names = scrapy.Field()
+    language_code = scrapy.Field()
+    country_code = scrapy.Field()
+    currency = scrapy.Field()
+    price = scrapy.Field()
+    image_urls = scrapy.Field()
+    description = scrapy.Field()
+    color = scrapy.Field()
+    sizes = scrapy.Field()
+    availability = scrapy.Field()
+
+    @classmethod
+    def create(cls, response, id, url, categories):
```

Comment:
We don't need such functions with Item classes. They are available by default. 
item = ProductItem(field_1="value1", ...)
However, if you want to extract some values from response, you can create a function in utils.